examples/react-native/metro.config.js

```diff
@@ -8,6 +8,9 @@ const INTERNAL_DEPENDENCY_FLAG = '*';
 // only 'packages' for now, but can be extended for use with other directories
 const INTERNAL_DIRECTORY_ROOTS = ['packages'];
 
+// internal packages used by the RN example
+const INTERNAL_USED_PACKAGES = ['ui', 'react-core', 'react-native'];
```

Comment:
Tiny random nit: what do you think about just `INTERNAL_PACKAGES`?

Reply:
Semantically that would include `vue`, `angular` and `react` right?

Reply:
That's fair, I was thinking as "Internal Packages" as packages customers do not consume directly. I'm good to keep this name to be explicit then.

Reply:
Oh i get it, yeah i would be more concerned if this was not inside a config file that is already using similar language